### PR TITLE
[Tests-Only] Fix duplicate scenario names and remove duplicate scenarios

### LINKED ISSUE #38577: [Tests-Only] Fix acceptance test scenarios with same title
### Description
Currently, there are many scenarios with the same title on the same file.
All scenarios should be unique and have different titles from each other. this should be fixed.

### PR DESCRIPTION
**Pull request by dpakach**
## Description

Fix duplicate scenario names and remove duplicate scenarios

## Related Issue

- Fixes https://github.com/owncloud/core/issues/38577

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by kiranparajuli589** [comment]
lgtm :+1:

File: tests/acceptance/features/apiProvisioning-v1/getUsers.feature [hunk 1]
```diff
@@ -8,7 +8,7 @@ Feature: get users
     Given using OCS API version "1"
 
   @smokeTest @notToImplementOnOCIS
-  Scenario: admin gets all users
+  Scenario: admin gets all users where admin user exists
```
Comment (jasson99) [inline tests/acceptance/features/apiProvisioning-v1/getUsers.feature:11]: This was a bit confusing for me, as admin user always exists, and the title does not specify about including admin user in the response.
 I would suggest something like:
```suggestion
  Scenario: admin gets all users and checks for all the users including the admin user
```

File: tests/acceptance/features/webUIMoveFilesFolders/moveFiles.feature [hunk 2]
```diff
@@ -36,7 +36,7 @@ Feature: move files
     And file "data.zip" should be listed on the webUI
 
   @skipOnFIREFOX
-  Scenario: move a file into a folder where a file with the same name already exists
+  Scenario: move a file into a folder where a file with the same name already exists where filename contains special characters
```
Comment (jasson99) [inline tests/acceptance/features/webUIMoveFilesFolders/moveFiles.feature:39]:
```suggestion
  Scenario: move a file into a folder where a file with the same name already exists and the filename contains special characters
```

**Review by jasson99 — COMMENTED** [review]
LGTM overall

File: tests/acceptance/features/apiSharePublicLink2/updatePublicLinkShare.feature [hunk 3]
```diff
@@ -379,7 +379,7 @@ Feature: update a public link share
       | 2               | 200             |
 
 
-  Scenario Outline: Adding public upload to a read only shared folder as recipient is not allowed using the old public API
+  Scenario Outline: Adding public link with all permisssions to a read only shared folder as recipient is not allowed using the old public API
```
Comment (phil-davis) [inline tests/acceptance/features/apiSharePublicLink2/updatePublicLinkShare.feature:382]:
```suggestion
  Scenario Outline: Adding public link with all permissions to a read only shared folder as recipient is not allowed using the old public API
```

File: tests/acceptance/features/apiSharePublicLink2/updatePublicLinkShare.feature [hunk 4]
```diff
@@ -398,7 +398,7 @@ Feature: update a public link share
       | 2               | 404              |
 
   @issue-ocis-reva-11
-  Scenario Outline: Adding public upload to a read only shared folder as recipient is not allowed using the new public API
+  Scenario Outline: Adding public link with all permisssions to a read only shared folder as recipient is not allowed using the new public API
```
Comment (phil-davis) [inline tests/acceptance/features/apiSharePublicLink2/updatePublicLinkShare.feature:401]:
```suggestion
  Scenario Outline: Adding public link with all permissions to a read only shared folder as recipient is not allowed using the new public API
```

File: tests/acceptance/features/apiSharePublicLink2/updatePublicLinkShare.feature [hunk 5]
```diff
@@ -417,7 +417,7 @@ Feature: update a public link share
       | 2               | 404              |
 
 
-  Scenario Outline: Adding public upload to a shared folder as recipient is allowed with permissions using the old public API
+  Scenario Outline: Adding public link with all permisssions to a read only to a shared folder as recipient is allowed with permissions using the old public API
```
Comment (phil-davis) [inline tests/acceptance/features/apiSharePublicLink2/updatePublicLinkShare.feature:420]:
```suggestion
  Scenario Outline: Adding public link with all permissions to a read only shared folder as recipient is allowed with permissions using the old public API
```

File: tests/acceptance/features/apiSharePublicLink2/updatePublicLinkShare.feature [hunk 6]
```diff
@@ -436,7 +436,7 @@ Feature: update a public link share
       | 2               | 200             |
 
   @issue-ocis-reva-11
-  Scenario Outline: Adding public upload to a shared folder as recipient is allowed with permissions using the new public API
+  Scenario Outline:  Adding public link with all permisssions to a read only folder as recipient is allowed with permissions using the new public API
```
Comment (phil-davis) [inline tests/acceptance/features/apiSharePublicLink2/updatePublicLinkShare.feature:439]:
```suggestion
  Scenario Outline:  Adding public link with all permissions to a read only folder as recipient is allowed with permissions using the new public API
```

**Review by phil-davis — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
I accidentally "Approved" the first time. So now I committed all the review suggestions. Now it looks good.

**Pull request merged**